internal/services/compute/orchestrated_virtual_machine_scale_set_resource.go

```diff
@@ -92,6 +92,35 @@ func resourceOrchestratedVirtualMachineScaleSet() *pluginsdk.Resource {
 				ValidateFunc: computeValidate.OrchestratedVirtualMachineScaleSetSku,
 			},
 
+			"sku_profile": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Given this is `ForceNew` what is the default value here? Are we going to trigger replacements for existing resources?